apps/pie-docs/src/_includes/nav.njk

```diff
@@ -2,11 +2,11 @@
 {% set navigationIcons = {
     Accessibility: 'PersonalityTest',
     'All about PIE': 'pie',
-    'Designers': 'bulb-lightning',
+    designers: 'bulb-lightning',
     Foundations: 'foundations',
```

Comment:
Probably a follow-up task, but I think we should convert all keys to lowercase for consistency.

Reply:
Yep – I'm going to raise this as a separate PR once this is merged. Wanted to keep this one a bit smaller 😄